ManageWikiSettings.php

```diff
@@ -462,6 +462,15 @@
 		'section' => 'edit',
 		'help' => 'Restrict {{DISPLAYTITLE}} to titles that normalize to the same canonical database key. Wikis with NoTitle extension installed have this config unset.',
 	],
+	'wgDisplayTitleHideSubtitle' => [
+		'name' => 'Don\'t display the page\'s original title below the display title',
+		'restricted' => false,
+		'from' => 'displaytitle',
+		'type' => 'check',
+		'overridedefault' => true,
```

Comment:
Should be false to match the default in ls

Reply:
Done